app.js

```diff
@@ -1,10 +1,145 @@
 /* Imports */
-// import { getRandomItem } from './utils.js';
+import { getRandomItem } from './utils.js';
 
 /* State */
+// let shell = ['shell-1', 'shell-2', 'shell-3'];
+let pearl = ['pearl-1', 'pearl-2', 'pearl-3'];
+let guess = ['guess1', 'guess2', 'guess3'];
+const display1 = document.getElementById('display-1');
+const display2 = document.getElementById('display-2');
+const display3 = document.getElementById('display-3');
 
-/* Actions */
-function loadPage() {}
+const guess1 = document.getElementById('guess-1');
+const guess2 = document.getElementById('guess-2');
+const guess3 = document.getElementById('guess-3');
+const shell1 = document.getElementById('shell-1');
+const shell2 = document.getElementById('shell-2');
+const shell3 = document.getElementById('shell-3');
+const playAgainButton = document.getElementById('play-again-button');
+// const totalDisplay = document.getElementById('total-display');
+// const winsDisplay = document.getElementById('wins-display');
+// const lossesDisplay = document.getElementById('losses-display');
+// let gameState = '';
+let computer = '';
+
+// let guess = [guess1, guess2, guess3];
+
+function loadPage() {
+    displayShells();
+    pearl.classList.remove('hidden');
+    // gameState = 'guess';
+}
+
+guess1.addEventListener('click', () => {
+    guess = guess1;
+    // gameState = 'results';
+    shell1.classList.add('reveal');
+    displayGuess();
+    handleResults();
+    displayScoreboard();
+});
+
+guess2.addEventListener('click', () => {
+    guess = guess2;
+    // gameState = 'results';
+    shell2.classList.add('reveal');
+    displayGuess();
+    handleResults();
+    displayScoreboard();
+});
+
+guess3.addEventListener('click', () => {
+    guess = guess3;
+    // gameState = 'results';
+    shell3.classList.add('reveal');
+    displayGuess();
+    handleResults();
+    displayScoreboard();
+});
+
+// function displayShells() {
+//     if (gameState = 'guess') {
+// pearl1.classList.add('hidden');
+// pearl2.classList.add('hidden');
+// pearl3.classList.add('hidden');
+function displayShells() {
+    // if ((gameState = 'guess')) { {
```

Comment:
I think you want this ```if``` check in there, because when the ```gameState``` switches to results, you'll want the shells / pearls to have the corresponding classes that affect where/whether they're on the page